tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/fsm/EmbeddedClassifier.scala

```diff
@@ -96,30 +85,58 @@ class DTCostFunctionTrainer(
       trainingVectorSource.groupVectorIteratorsByTask
 
     var progressCounter = 1
-    (taskTrainingVectors map {
-      case (task, trainingVectorIter) => {
-        val trainingVectors = trainingVectorIter.toIterable
-        println(s"Task ${progressCounter} of ${trainingVectorSource.tasks.size}" +
-          s"(${task.filenameFriendlyName}) has ${trainingVectors.size} training vectors")
+    var taskClassifierSeq: Seq[(ClassificationTask, File)] = Seq() // TODO: do we need a var here?
```

Comment:
After creating it, the next thing that happens is `taskClassifierSeq.toMap mapValues { ... }`, which loses the `ClassificationTask` instances.